deploy/GOV/001-GovernanceFactory.ts

```diff
@@ -0,0 +1,69 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction, DeployResult } from "hardhat-deploy/types";
+
+/**
+ * @title Governance Factory Deployment Script
+ * @dev Deploys the governance factory contract and links the required libraries.
+ * Usage: `npx hardhat deploy --network <NETWORK> --tags GOV_DEPLOY`
+ */
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, deployments, getNamedAccounts, getChainId } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+
+  const CELO_REGISTRY = process.env.CELO_REGISTIRY_ADDRESS;
```

Comment:
Minor point, but this can be a constant. The registry is the same on all chains 0x000000000000000000000000000000000000ce10